packages/main/src/components/AnalyticalTable/scrollbars/VerticalScrollbar.tsx

```diff
@@ -13,10 +15,35 @@ interface VerticalScrollbarProps {
   classNames: ClassNames;
 }
 
+const isChrome = isChromeFn();
```

Comment:
Could lead to an hydration error in SSR environments. This returns always false on the server, on some clients it might be true -> Hydration Error